portworx/values.yaml

```diff
@@ -35,8 +35,8 @@ repo:
   dr: "709825985650.dkr.ecr.us-east-1.amazonaws.com/portworx/dr"
 
 versions:
-  operator: "1.4.2"
-  enterprise: "2.6.2.1"
-  ociMon: "2.6.2.1"
-  autoPilot: "1.3.0"
-  stork: "2.6.1"
+  operator: "1.6.0"
+  enterprise: "2.8.1"
+  ociMon: "2.8.1"
+  autoPilot: "1.3.1-rc1"
```

Comment:
We need to change this before we publish it, right?

Reply:
we will yes. i have pushed images and tested the install. it works
but i want to make sure we pass CVE's first